dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 build~=0.1.0
-black~=22.0
+black~=24.3
```

Comment:
The newer 24.4.0 is available:
https://pypi.org/project/black/24.4.0/

Reply:
Nice. The lower bound is only there because of the vulnerability though. It's not supposed to be set to the latest version.